controllers/apply/form-router-next/lib/file-uploads.js

```diff
@@ -5,11 +5,12 @@ const AWS = require('aws-sdk');
 const get = require('lodash/get');
 const keyBy = require('lodash/keyBy');
 const mapValues = require('lodash/mapValues');
+const NodeClam = require('clamscan');
 
-const { isTestServer } = require('../../../../common/appData');
+const { isTestServer, isDev } = require('../../../../common/appData');
 const { S3_KMS_KEY_ID } = require('../../../../common/secrets');
 const logger = require('../../../../common/logger').child({
-    service: 's3-uploads'
+    service: 'file-uploads'
```

Comment:
Just a note to remind us to update the dashboard(s) when this goes live as the metrics there will refer to the old name.